web/html/images/image-height-width.md

```diff
@@ -55,11 +54,10 @@ img {
 
 Fill in the appropriate attributes/values for creating an image in HTML with dimensions 300x200.
 
-`<??? ???="image.svg" ???="200" ???="300">`
+`<??? ???="image.svg" height="200" ???="300">`
```

Comment:
Why did we remove `???` for height? The question says dimensions are `300x200` so it's clear only height can be 200.

Reply:
Because people would still make a mistake and then blame the app and not their own inability to properly process the question

Reply:
The question was first wrong when created, then updated to be like it is now(assuming people would clearly understand which dimension goes where), however, that wasn't the case. So now its 100% "mistake-proof"